packages/block-library/src/footnotes/format.js

```diff
@@ -62,6 +63,16 @@ export const format = {
 			return null;
 		}
```

Comment:
Could we not simply check for the presence of a post id? Would there be a post id in context when you select inside a template part?

Reply:
Oh, ignore me, I thought this was for template parts.